torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8836,9 +8843,19 @@ def _generate_sample_inputs_nn_loss(op_info, device, dtype, requires_grad, **kwa
 
 def sample_inputs_hinge_embedding_loss(op_info, device, dtype, requires_grad, **kwargs):
     for input, target, d in _generate_sample_inputs_nn_loss(op_info, device, dtype, requires_grad, **kwargs):
+        # target should contain either 1 or -1 as per docs
+        with torch.no_grad():
```

Comment:
Was this no grad context required? @Lezcano  -- maybe an example of how our attempt to produce samples in a no grad context is imperfect?

Reply:
if it was, we should figure out what's going on there...

Reply:
Works without `torch.no_grad` as well. I saw it at a couple of places in the file and played it safe here.  (will remove)